.github/workflows/RedeployInferenceACI.yml

```diff
@@ -0,0 +1,74 @@
+name: GitHub Actions Deploy AI4Orca ML ACI
+on:
+  # Triggers the workflow on merges only
+  push:
+    branches: [ main ]
+    paths:
+      - "InferenceSystem/configProduction/**"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
+      - run: echo "🖥️ The workflow is now ready to test your code on the runner."
+      - name: List files in the repository
+        run: |
+          ls ${{ github.workspace }}
+      - name: 'Login via Azure CLI'
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - name: 'Deploy AI4Orca to Azure Container Instances-orcasound-lab-live'
```

Comment:
Consider breaking these aci-deploy steps out into a separate job that employs the strategy.matrix functionality of a job to reduce repetition. https://docs.github.com/en/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix